site/docs/v1/tech/apis/identity-providers/_openid-connect-request-body.adoc

```diff
@@ -71,18 +71,18 @@ The client authentication method to use with the OpenID Connect identity provide
 
 ifdef::update[]
 [field]#identityProvider.oauth2.emailClaim# [type]#[String]# [required]#Required# [since]#Available since 1.17.3#::
-An optional configuration to modify the expected name of the claim returned by the IdP that contains the email address. The Access Token is inspected for this claim first. If it is not present and an Id Token is available, the Id Token is examined as well.
+An optional configuration to modify the expected name of the claim returned by the IdP that contains the email address. The UserInfo response found using the Access Token is inspected for this claim first. If it is not present and an Id Token is available, the Id Token is examined as well.
 endif::[]
 ifndef::update[]
 [field]#identityProvider.oauth2.emailClaim# [type]#[String]# [optional]#Optional# [default]#defaults to `email`# [since]#Available since 1.17.3#::
-An optional configuration to modify the expected name of the claim returned by the IdP that contains the email address. The Access Token is inspected for this claim first. If it is not present and an Id Token is available, the Id Token is examined as well.
+An optional configuration to modify the expected name of the claim returned by the IdP that contains the email address. The UserInfo response found using the Access Token is inspected for this claim first. If it is not present and an Id Token is available, the Id Token is examined as well.
```

Comment:
I don't think there is a difference between create and update, we always default to `email`.